static/app/gettingStartedDocs/node/awslambda.tsx

```diff
@@ -10,71 +14,177 @@ import {
   getCrashReportModalConfigDescription,
   getCrashReportModalIntroduction,
 } from 'sentry/components/onboarding/gettingStartedDoc/utils/feedbackOnboarding';
+import {AwsLambdaArn} from 'sentry/gettingStartedDocs/node/awslambdaArnSelector';
 import {t, tct} from 'sentry/locale';
-import {trackAnalytics} from 'sentry/utils/analytics';
 import {
   getInstallConfig,
   getNodeProfilingOnboarding,
   getSdkInitSnippet,
 } from 'sentry/utils/gettingStartedDocs/node';
-import {
-  InstallationMode,
-  platformOptions,
-} from 'sentry/views/onboarding/integrationSetup';
+
+export enum ModuleFormat {
+  CJS = 'cjs',
+  ESM = 'esm',
+}
+
+export const platformOptions = {
```

Comment:
thanks for adding these! Can these platform option toggles also contain a tooltip (like feature selection toggles)? Would be cool if we could explain add a 1-liner like

- "If you run in or transpile your function to CJS ("require" syntax), use this option" 
- "If you run your lambda in ESM ("import" syntax), use this option"

If not possible feel free to disregard/defer for later :)